src/corner-dialog/src/CornerDialog.js

```diff
@@ -148,7 +148,7 @@ export default class CornerDialog extends PureComponent {
     onCancel: close => close(),
     onConfirm: close => close(),
     onCloseComplete: () => {},
-    position: AbsolutePosition.BOTTOM_LEFT
+    position: 'BOTTOM_RIGHT'
```

Comment:
We should use the same constants that we export from Evergreen – that means importing `Position` and assigning the default as `Position.BOTTOM_RIGHT` instead of a string literal here.